plugins/plugin-optimize/test/__snapshots__/plugin.test.js.snap

```diff
@@ -0,0 +1,26 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`@snowpack/plugin-webpack minimal - all options: console.log 1`] = ``;
+
+exports[`@snowpack/plugin-webpack minimal - all options: fs.writeFileSync 1`] = `
+FILE: stubs/minimal/index.html
+<!DOCTYPE html><html lang="en"><head><meta charset="UTF-8"><title>Minimal test</title><link rel="stylesheet" href="style.css"></head><body><script type="module" src="index.js"></script></body></html>
```

Comment:
@drwpow I'm surprised that the HTML file is optimized, in the test I set `minifyHTML: false`. Is this a bug or am I missing something?

Reply:
🤷 sounds like a bug! Can you take a look and fix as a part of this PR?